### ISSUE TITLE
Додати більше різноманітної інформації про Україну на сайт

### ISSUE THREAD

**Issue by nagornyi** [Kraiany/ukraine-info#17]
http://ukuraina-jouhou.info/
Потрібно більше інформації про міста, історію, культуру, цікаві місцевості, факти про Україну і т.п. Все можна залишати тут у коментарях, адміни проекту потім додадуть це все на сайт до існуючих, або нових сторінок, якщо це нова категорія.

**Comment by ukrainejapan** [comment]
Думка-ідея щодо оформлення:усі наявні міста об'єднати в єдину категорію, а при переході в цю категорію - перелік міст з посиланнями на їх сторінки в, наприклад, ウクライナの地理 (Географія України) або ウクライナの地域 (регіони України)
Внизу посилання на 在日ウクライナ人コミュニティ「クラヤニイ」(Краяни, Kraiany)のプロジェクトです, можливо, краще поставити на сторінку https://www.facebook.com/ukrainians.japan , де вже є посилання на групу, а то при переході на групу - нічого не побачиш, адже вона закрита.

**Comment by nagornyi** [comment]
@ukrainejapan так, згоден, треба було б, бо кількість міст буде збільшуватись з часом і сторінка розростатиметься. Як буде ще трохи вільного часу, спробую зробити потрібний layout для цього. Можна наприклад зробити категорії ウクライナの地域 та ウクライナの自然. в першій категорії будуть міста, в другій - гори, річки, море і т.п.
Вже змінив посилання на відкриту групу.

**Comment by ukrainejapan** [comment]
Отримали пропозицію використовувати світлини фотографа Макса Соломко https://www.facebook.com/max.d.solomko (член Союзу фотохудожників України) www.maxsolomko.com . Деякі фото є на сайті, або можна звернутись до нього з питаннями, які фото нам необхідні.
Зараз можна вибрати міста, яких ще нема в переліку і створювати їх сторінки, як мінімум - з фото.

**Comment by nagornyi** [comment]
Дякую, на його сайті є гарні фото Києва, на вихідних закину на сайт з копірайтом.

**Comment by ukrainejapan** [comment]
На сторінку про Карпати можна додати підпункт, тощо. Текст оригінальний підправлений носієм.
https://www.facebook.com/tuaukraine/posts/561219097311849
